plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -44,7 +56,64 @@ public void launch(ILaunchConfiguration configuration, String mode, ILaunch laun
     TargetPlatform.showConsole(console);
     console.clearConsole();
 
+    LocalAppEngineServerDelegate serverDelegate = LocalAppEngineServerDelegate.getAppEngineServer(server);
+    int debugPort = -1;
+    if (ILaunchManager.DEBUG_MODE.equals(mode)) {
+      final int port = getDebugPort();
```

Comment:
This code has previously been in an `IServerChangeListener` (or something to that effect).  So it can be dropped